packages/migrate/src/__tests__/DbPull/sqlite.test.ts

```diff
@@ -413,15 +413,11 @@ describe('common/sqlite', () => {
     const result = DbPull.new().parse(['--schema=./prisma/invalid.prisma', '--force'])
     await expect(result).resolves.toMatchInlineSnapshot(`""`)
 
-    expect(captureStdout.getCapturedText().join('\n')).toMatchInlineSnapshot(`
+    expect(captureStdout.getCapturedText().join('')).toMatchInlineSnapshot(`
```

Comment:
Joining with `\n` is not quite correct here - we capturing `process.stdout.write` and that does not automatically add a newline at the end.